apps/desktop-wallet/src/hooks/useHookCoordinates.ts

```diff
@@ -0,0 +1,39 @@
+import { useCallback, useRef, useState } from 'react'
```

Comment:
I wonder if we could put this in the `features`  folder somewhere.
Also, we could name the hook `useElementAnchorCoordinates` and/or at least rename "hook" into "anchor" to avoid confusion. WDYT?

Reply:
Fine by me! I'll look where I can fit this in the features

Reply:
Since it's used in tandem with the Popup component, I moved it in there.